modules/gapi/src/streaming/onevpl/accelerators/accel_policy_dx11.cpp

```diff
@@ -283,12 +293,24 @@ mfxStatus VPLDX11AccelerationPolicy::on_alloc(const mfxFrameAllocRequest *reques
         desc.BindFlags = 0;
     }
 
+    /* NB:
+     * On the one hand current OpenVINO API doesn't support texture array and
+     * D3D11 API doesn't allow to address specific texture element in array.
+     * On the other hand using textures array should be more performant case
+     * in applications (according to community experience)
+     * So, to be compliant with OV let's turn off textures array feature, but keep
+     * this code in commented section to consider such "optimization" in future
+     */
+    /*
     size_t main_textures_count = 1;
     if (D3D11_BIND_RENDER_TARGET & desc.BindFlags) {
         GAPI_LOG_DEBUG(nullptr, "Use array of testures instead of texture array");
```

Comment:
complicated question:
-there are **array** of textur**es** like as: `std::vector<Texture>`
-and on the other side **texture array** where `Texture` holds subtextures by itself
```
// single texture with subresources
    desc.ArraySize = request->NumFrameSuggested;
```
The second one is turned off in current approach, because  OV use textures caching in their implementation inside and that caching techniques require different textures pointer and doens't works with single texture with subresources array